__tests__/api.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+const db = require("../db/connection");
+const { selectAllTopics } = require("../models/topics.model");
+
+beforeEach(() => {
+    return seed(testData);
+    });
+    afterAll(() => {
```

Comment:
Please remember to try and format your code in a nice and readable way that adheres to the formatting guides for JS.